config/routes/api.js

```diff
@@ -1,11 +1,40 @@
 var express             = require('express')
   , errorHandler        = require('express-error-middleware')
+  , multer              = require('multer')
+  , path                = require('path')
+  , crypto              = require('crypto')
   , helpers             = require('./helpers')
   , ensureAuthenticated = helpers.ensureAuthenticated
   , articles            = require('../../app/controllers/articles')
   , unions              = require('../../app/controllers/unions');
 
+var EXTENSION_LOOKUP = {
```

Comment:
this could definitely be more clever, but extension guessing is pretty dangerous so I think it might be a good middleground